pkg/scanner/scanner.go

```diff
@@ -122,16 +123,19 @@ func (s *Scanner) initScan() error {
 		for _, container := range pod.Status.ContainerStatuses {
 			containerNameToImageId[container.Name] = container.ImageID
 		}
+		for _, container := range pod.Status.InitContainerStatuses {
+			containerNameToImageId[container.Name] = container.ImageID
+		}
 
-		for _, container := range pod.Spec.Containers {
+		_ = k8s_utils.VisitContainersWithPath(&pod.Spec, func(container *corev1.Container, _ *field.Path) error {
```

Comment:
You can create one slice from pod.Spec.Containers & pod.Spec.InitContainers and go over it - why the need to use `VisitContainersWithPath`?